tests/utils_tests/test_datastructures.py

```diff
@@ -102,6 +102,12 @@ def test_multivaluedict(self):
         self.assertEqual(d.getlist("lastname"), ["Holovaty", "Willison"])
         self.assertEqual(list(d.values()), ["Simon", "Developer", [], "Willison"])
 
+        d.setlistdefault("lastname", ["Doe"])
+        self.assertEqual(d.getlist("lastname"), ["Holovaty", "Willison"])
```

Comment:
I added this assert to show that if `key` already exists it won't be overwritten. But with the test below would be enough to complete the coverage.